subprojects/core/src/main/java/org/gradle/plugin/management/internal/autoapply/AutoAppliedBuildScanPlugin.java

```diff
@@ -28,5 +28,5 @@
     PluginId ID = new DefaultPluginId("com.gradle.build-scan");
     String GROUP = "com.gradle";
     String NAME = "build-scan-plugin";
-    String VERSION = "1.11";
+    String VERSION = "1.12";
```

Comment:
I guess we should fix the test: https://builds.gradle.org/viewQueued.html?itemId=10934610&tab=queuedBuildOverviewTab

Seem like the URL for the license changed from `ttps://scans.gradle.com/terms-of-service` to `https://gradle.com/terms-of-service`.

Reply:
This test failure seems to point to a problem with the plugin: https://builds.gradle.org/viewLog.html?buildId=10941268&tab=buildResultsDiv&buildTypeId=Gradle_Check_Stage_QuickFeedbackLinuxOnly_Trigger

WDYT?

Reply:
Yes, that's a legit failure. We're reusing validation logic from the non-interactive path. We'll fix it and let you know when it's ready. Thanks for having a look!